nuget/MacMobileCenter.nuspec

```diff
@@ -11,18 +11,23 @@
     <projectUrl>https://aka.ms/telgml</projectUrl>
     <licenseUrl>https://aka.ms/vbgfx2</licenseUrl>
     <iconUrl>https://aka.ms/p2wifj</iconUrl>
+    <dependencies>
+        <group targetFramework="netstandard1.0">
+            <dependency id="NETStandard.Library" version="1.6.0" />
```

Comment:
It is compatible with .NET Standard 1.0. See https://www.nuget.org/packages/NETStandard.Library/